packages/scripts/src/commands/app-data/convert/processors/flowParser/flowParser.ts

```diff
@@ -67,6 +67,11 @@ export class FlowParserProcessor extends BaseProcessor<FlowTypes.FlowTypeWithDat
    */
   public async postProcess(flows: FlowTypes.FlowTypeWithData[]): Promise<IParsedWorkbookData> {
     const flowsByType: IParsedWorkbookData = groupJsonByKey(flows, "flow_type");
+    for (const [flowType, parser] of Object.entries(this.parsers)) {
```

Comment:
This feels sensible to me, can see the function comment asserts that postprocessing methods should be called by flowtype but appears to be omitted. 

What I think would be most useful is trying to capture some documentation/flowchart on the processing order. I think it should actually be fairly straight-forward as the processing order is just something along the lines of

- Apply xlsx converter to extract flows from xlsx files.
- Apply default parser by flow, applying any `@default` or `@row` statements, convert _list and _collection columns, removes meta fields, handles translated fields, processes nested groups etc.
- Apply methods from extended parsers (e.g. data_list, data_pipe). These assign overrides per row, per flow, and across all processed flows

There is also a side flow that explains how caching and queues are used, but again reasonably straightforward (generally tracked by file md5 checksum, so that if a file contents are unchanged cached results used; exceptions possible, currently used for data_pipes which depend on child inputs so cache ignored)

If not creating now, might be good to ensure an issue is created (possibly linking to this comment for some reference)